samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -5,16 +5,42 @@
 from textwrap import dedent
 from typing import Any, Dict, List, Optional
 
+import logging
```

Comment:
Please move non-required imports into command execution block so that running `--help` won't need to import any of these. See an example here; https://github.com/aws/aws-sam-cli/blob/develop/samcli/commands/sync/command.py#L244

Reply:
Looking at the import. I don't think it's needed. Will remove it. Thanks for catching it.

Reply:
Not at the moment. We used to have something in development guide or in contributing but couldn't find it now.

Reply:
OK it was in design document; 
> Each command, along with any subcommands, is implemented using Click annotations. They are not directly wired with the core of the CLI. Instead, commands are dynamically loaded into the CLI at run time by importing the Python package implementing the command. For example, assuming two commands are implemented at Python packages foo.cli.cmd1 and foo.cli.cmd2, then the CLI framework will dynamically import these two packages and connect them to parent Click instance. The CLI framework expects the command's Click object to be exposed through an attribute called cli.

https://github.com/aws/aws-sam-cli/blob/develop/DESIGN.md